go-controller/pkg/network-controller-manager/node_network_controller_manager.go

```diff
@@ -57,19 +56,19 @@ func (ncm *nodeNetworkControllerManager) CleanupDeletedNetworks(allControllers [
 
 // newCommonNetworkControllerInfo creates and returns the base node network controller info
 func (ncm *nodeNetworkControllerManager) newCommonNetworkControllerInfo() *node.CommonNodeNetworkControllerInfo {
-	return node.NewCommonNodeNetworkControllerInfo(ncm.client, ncm.watchFactory, ncm.recorder, ncm.name, ncm.isOvnUpEnabled)
+	return node.NewCommonNodeNetworkControllerInfo(ncm.ovnNodeClient.KubeClient, ncm.ovnNodeClient.AdminPolicyRouteClient, ncm.watchFactory, ncm.recorder, ncm.name, ncm.isOvnUpEnabled)
 }
 
 // NewNodeNetworkControllerManager creates a new OVN controller manager to manage all the controller for all networks
 func NewNodeNetworkControllerManager(ovnClient *util.OVNClientset, wf factory.NodeWatchFactory, name string,
 	eventRecorder record.EventRecorder) (*nodeNetworkControllerManager, error) {
 	ncm := &nodeNetworkControllerManager{
-		name:         name,
-		client:       ovnClient.KubeClient,
-		Kube:         &kube.Kube{KClient: ovnClient.KubeClient},
-		watchFactory: wf,
-		stopChan:     make(chan struct{}),
-		recorder:     eventRecorder,
+		name:          name,
+		ovnNodeClient: &util.OVNNodeClientset{KubeClient: ovnClient.KubeClient, AdminPolicyRouteClient: ovnClient.AdminPolicyRouteClient},
```

Comment:
shouldnt you just pass in ovnClient here and  set it. Then in NewOVNClientset you add the AdminPolicyRouteClient part when you craete the clientset?